uiowa/uiowa
# [INRC] Events

Resolves https://github.com/uiowa/uiowa/issues/8007.




# How to test

```
 ddev blt ds --site=inrc.law.uiowa.edu  && ddev drush @lawinrc.local uli /node/221/edit
```
- Scroll to the bottom and add Cost, Event sponsor, and Contact information. 
- Save the page and confirm that the display is similar to the screenshot below. 
<img width="1406" alt="Screenshot 2024-09-04 at 11 29 22 AM" src="https://github.com/user-attachments/assets/afa0e31a-cbfb-4373-a0e9-5094f630c179">
